src-front/src/components/WordCloud/WordCloud.tsx

```diff
@@ -154,17 +154,38 @@ export default function WordCloud({
   const [inputValue, setInputValue] = useState("");
   //const [error, setError] = useState("");
   const [scores, setScores] = useState<Record<string, number>>({});
-  const [wordsBestProgress, setWordsBestProgress] = useState<
-    Record<string, number>
-  >({});
+  // const [wordsBestProgress, setWordsBestProgress] = useState<
```

Comment:
Is support for displaying global best progress dropped?

Reply:
Idk I thought I wasn't coded in the back, wordsBestProgress is always empty on my side

Reply:
It is supposed to be fully functional, if not it is a bug.

Reply:
see: #154